src/main/kotlin/no/nav/helse/fritakagp/integration/g.kt

```diff
@@ -8,29 +8,21 @@ import java.time.Duration
 import java.time.LocalDate
 
 class GrunnbeløpClient(val httpClient: HttpClient) {
-    val cache = SimpleHashMapCache<GrunnbeløpInfo>(Duration.ofDays(1), 2)
-
-    fun hentGrunnbeløp(): GrunnbeløpInfo {
-        if (cache.hasValidCacheEntry("g")) return cache.get("g")
-
-        val g = runBlocking {
-            httpClient.get<GrunnbeløpInfo> {
-                url("https://g.nav.no/api/v1/grunnbeløp")
-            }
+    fun hentGrunnbeløp(dato: LocalDate): GrunnbeløpInfo = runBlocking {
+        httpClient.get<GrunnbeløpInfo> {
```

Comment:
Stemmer det ikke at grunnbeløpet justeres 1. mai hvert år? Da er det vel fremdeles vært mulig å cache med nøkler basert på inputdato. F. eks. kan vi bruke nøkkel "2021-05" for `2022-05-01 > dato >= 2021-05-01`.

Reply:
Ah, visste ikke at det blir justert på en fast dato hvert år. Det muliggjør cache ja

Reply:
Det står 1. mai på https://www.nav.no/no/nav-og-samfunn/kontakt-nav/utbetalinger/grunnbelopet-i-folketrygden